### ISSUE TITLE
Inception Score on UCF101

### ISSUE THREAD

**Issue by vladyushchenko** [pfnet-research/tgan#11]
Hi, 
nice work. I have a small question about the inception score evaluation procedure.
As referenced in https://github.com/pfnet-research/tgan/issues/8 
the normalization file for C3D network was not provided with the code.
The proposed solution has a file, that was calculated for image resolution 112 * 112, 
while you are using 128 * 171.

Could you, please, explain how it was calculated?
Is it taken from .binaryproto normalization file of Caffe C3D model, finetuned on UCF
https://github.com/facebook/C3D/tree/master/C3D-v1.0/examples/c3d_finetuning
or the initial model trained on Sports-1M
https://github.com/facebook/C3D/tree/master/C3D-v1.0/examples/c3d_feature_extraction ?
Do you perform any additional transformations?

Thank you in advance,
Vlad

**Comment by rezoo** [comment]
Hello Vlad,

Sorry for replying late.
We calculated the inception score with the pre-trained model trained on Sports-1M dataset and fine-tuned on UCF101. The model can be downloaded at [this URL](http://vlg.cs.dartmouth.edu/c3d/).
To compute the inception score, we resized the video into 128x128px, and sent it into the above pre-trained model.

Please reply or ask me directly via email if you could have some additional questions.

Sincerely,
Saito

**Comment by VladYushchenko** [comment]
Dear Saito,

thank you for the reply. I was able to get the reported score for both
TGAN(Normal GAN) and TGAN(SVC) you reported in the paper.

I did notice that you load normalization file 'mean2.npz' from disk, which
you saved previously in
"/mnt/sakura201/mitmul/codes/tgan2_orig/inception/mean2.npz".
This file is given in your code
<https://github.com/pfnet-research/tgan/blob/53fde1bcb59e2a57f76b9b26097226ec0423811f/evaluation/calc_inception.py#L35>
and you use it here.
<https://github.com/pfnet-research/tgan/blob/53fde1bcb59e2a57f76b9b26097226ec0423811f/evaluation/calc_inception.py#L60>

The only additional question I have is:

have you created the normalization file for ucf_101 dataset following this
instruction (Section II.A)
<https://docs.google.com/document/d/1-QqZ3JHd76JfimY4QKqOojcEaf5g3JS0lNh-FHTxLag/edit>
Or did you convert pre-calculated by C3D authors
train01_16_128_171_mean.binaryproto
<http://C3D/C3D-v1.0/examples/c3d_finetuning/train01_16_128_171_mean.binaryproto>
in you




чт, 6 груд. 2018 о 08:45 Masaki Saito <notifications@github.com> пише:

**Comment by vladyushchenko** [comment]
Closing issue, normalization file is converted from [C3D binaryproto](C3D/C3D-v1.0/examples/c3d_finetuning/train01_16_128_171_mean.binaryproto). 
Data is provided as plain array, but can be reshaped and saved as numpy array, as discussed in [C3D issues](https://github.com/facebook/C3D/issues/235)
